package.json

```diff
@@ -14,29 +14,34 @@
   "dependencies": {
     "@expo-google-fonts/didact-gothic": "^0.1.0",
     "@expo-google-fonts/poppins": "^0.1.0",
+    "@react-native-async-storage/async-storage": "1.13.4",
```

Comment:
I have not checked but please make sure you installed this package using expo add because I believe it's one of those packages that expo expects to be with a specific version to guarantee support

Reply:
I'll double check this!